package.json

```diff
@@ -1,53 +1,56 @@
 {
-  "name": "@azure-tools/python-codegen-monorepo",
-  "private": true,
-  "version": "0.1.0",
-  "description": "Monorepo for python codegen",
-  "workspaces": [
-    "packages/*"
-  ],
-  "scripts": {
-    "preinstall": "npx only-allow pnpm",
-    "watch": "tsc --build ./tsconfig.ws.json --watch",
-    "build": "pnpm -r run build",
-    "clean": "pnpm -r run clean",
-    "lint": "pnpm -r run lint",
-    "format": "npx prettier **/*.ts --write",
-    "check-format": "npx prettier **/*.ts --check",
-    "check-version-mismatch": "syncpack list-mismatches",
-    "fix-version-mismatch": "syncpack fix-mismatches"
-  },
-  "engines": {
-    "node": ">=16.0.0",
-    "pnpm": ">=7.5.2"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Azure/autorest.python.git"
-  },
-  "author": "Microsoft",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Azure/autorest.python/issues"
-  },
-  "homepage": "https://github.com/Azure/autorest.python#readme",
-  "devDependencies": {
-    "@typespec/prettier-plugin-typespec": "~0.46.0",
-    "eslint": "^8.44.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-unicorn": "^46.0.0",
-    "prettier": "^2.8.8",
-    "typescript": "~5.1.3",
-    "syncpack": "^9.8.6"
-  },
-  "syncpack": {
-    "dependencyTypes": [
-      "dev",
-      "overrides",
-      "peer",
-      "pnpmOverrides",
-      "prod",
-      "resolutions"
-    ]
+    "name": "@azure-tools/python-codegen-monorepo",
```

Comment:
nit: would roll back this formatting change

Reply:
formatting change reverted